### ISSUE TITLE
Lego Indiana Jones: The Original Adventures (32330)

### ISSUE THREAD

**Issue by sudoman281** [ValveSoftware/Proton#1990]
# Whitelist Request
- Name of the game to be whitelisted: Lego Indiana Jones: The Original Adventures
- Steam AppID of the game: 32330

## System Information
- GPU: nVidia GTX 1060
- Driver/LLVM version: 4.6.0 NVIDIA 410.73
- Kernel version: 4.19.1.arch1
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/sudoman281/98d82e05efae4c4743166c469e893317
- Proton version: 3.7-8

## I confirm:
- [ x ] that pressing the `Play` button in the Steam client is sufficient.
- [ x ] that [runtime config options](https://github.com/ValveSoftware/Proton#runtime-config-options)
      are necessary to run the game.
- [ x ] that no workarounds other than the mentioned ones are necessary.

## Issues
- [ ] I haven't experienced any issues.
- [ ] There are no issues left open for this game.
- [ x ] Although I consider the gaming experience equal to Windows there are
      remaining issues:
I had a problem with looping a part of the music in the menu, cutscenes, etc. all over again. I had to enable PROTON_NO_ESYNC in user_settings.py in order to make it work. Maybe this was necessary because I use JACK audio server, but I can't tell certainly.

**Comment by ghost** [comment]
> # Whitelist Request
> * Name of the game to be whitelisted: Lego Indiana Jones: The Original Adventures
> * Steam AppID of the game: 32330
> 
> ## System Information
> * GPU: nVidia GTX 1060
> * Driver/LLVM version: 4.6.0 NVIDIA 410.73
> * Kernel version: 4.19.1.arch1
> * Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/sudoman281/98d82e05efae4c4743166c469e893317
> * Proton version: 3.7-8
> 
> ## I confirm:
> * [ x ] that pressing the `Play` button in the Steam client is sufficient.
> * [ x ] that [runtime config options](https://github.com/ValveSoftware/Proton#runtime-config-options)
>   are necessary to run the game.
> * [ x ] that no workarounds other than the mentioned ones are necessary.
> 
> ## Issues
> * [ ]  I haven't experienced any issues.
> * [ ]  There are no issues left open for this game.
> * [ x ] Although I consider the gaming experience equal to Windows there are
>   remaining issues:
>   I had a problem with looping a part of the music in the menu, cutscenes, etc. all over again. I had to enable PROTON_NO_ESYNC in user_settings.py in order to make it work. Maybe this was necessary because I use JACK audio server, but I can't tell certainly.

Yes PROTON_NO_ESYNC=1 is needed to work out of the box but its not your JACK audio server

**Comment by jp7677** [comment]
~~With Proton 4.11 I'm no longer able to select the native resolution of my monitor (3440x1440). The game now falls back to lower resolutions. This happens with both Wine3D and D9VK. With Proton 4.2 I could choose 3400x1440.
This could be related to https://github.com/ValveSoftware/Proton/issues/2932~~

Edit: This has been fixed with Proton 4.11-2. Thanks a lot!

**Comment by sudoman281** [comment]
Tested with Proton 4.11 and still working absolutely fine.

**Comment by ranplayer** [comment]
I've found a regression for this game when using **Proton 10.0-X** instead of **9.0-4**: the initial controller button mapping does not load (you need to use your keyboard to manually map). This is also affecting [Lego Batman](https://github.com/ValveSoftware/Proton/issues/1156).

Proton 10.0-2 (RC) logs:  [steam-32330-proton10.tar.gz](https://github.com/user-attachments/files/21023160/steam-32330-proton10.tar.gz)

Proton 9.0-4 button mapping:

![Image](https://github.com/user-attachments/assets/51f4992c-15f0-4ee8-8d82-7ba9593aa792)

Proton 10.0-2 (RC) button mapping:

![Image](https://github.com/user-attachments/assets/2150d51f-d96f-4176-8a7d-ca80121800e0)

Specs:
- OS: Arch
- Kernel: 6.12.29
- GPU: RX 6700 XT (Mesa 25.0.5)
- CPU: Ryzen 7 5700G (8c/16t)
- Controller: DS4 (Steam input on)

**Comment by Sunie83** [comment]
I've tried a bunch of different proton versions, including 9, 8, 7 and 4 and several GE-Proton versions on two different linux based (fedora and debian) systems, but in the best case scenario this game now boots up with fully grey graphics, then severely garbled once in game. It is completely unplayable in this state. from what I can tell from [](https://www.protondb.com/app/32330), the game used to work until at least about 3 months ago with the proton versions I've tried.
